### PR TITLE
Simplify CoC page

### LINKED ISSUE #781: Update list of project report methods/emails
It seems that we should have the list of project reporting methods for every project on this list.

https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md

Related: #775

### PR DESCRIPTION
**Pull request by tobie**
Closes #781.
Closes #775.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,66 +1,43 @@
...
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
+
+
+## Report an issue in a project
+
+1. To report an issue in one of the projects listed below, please use the method provided. The project itself is responsible for managing these reports.
+    * **AMP Project:** <code-of-conduct@amp.dev>
+    * **Appium:** email maintainers
```
Comment (tobie) [inline CODE_OF_CONDUCT.md:17]: That needs to be fixed by at least providing a link

**Comment by tobie** [comment]
The linter is a bit excessive, here, imho, no?

**Review by eemeli — APPROVED** [review]

**Review by joesepi — APPROVED** [review]
LGTM. Thanks @tobie !

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,66 +1,43 @@
...
+    * **Dojo:** [Fill this reporting form](https://docs.google.com/forms/d/1ZiCUv3FsarWetoQ8J5QnCdIrHLj0xMmwNSLCyBkio7c/viewform?edit_requested=true)
+    * **Electron:** <coc@electronjs.org>
+    * **Express.js:** <express-coc@lists.openjsf.org>
+    * **Fastify:** <hello@matteocollina.com> or <tommydelved@gmail.com>
+    * **HospitalRun:** <hello@hospitalrun.io>
+    * **Node.js:** <report@nodejs.org>
+    * **Node-RED:** <team@nodered.org>
+    * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
+    * **Webhint:** <support@webhint.io>
+2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
```
Comment (mhdawson) [inline CODE_OF_CONDUCT.md:27]: This is an interesting addition which I'm not sure we've discussed. That the CPC is responsible for taking reports for projects that don't have a mechanism that makes sense to me but we might want to add as a responsibility in https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc.  We currently have this 
"Supporting projects in their enforcement of Code of Conduct violations and escalation." but what is captured by this line I think goes beyond that.

Comment (mhdawson) [reply CODE_OF_CONDUCT.md:27]: @openjs-foundation/cpc FYI for broader visibliity.

**Review by mhdawson — APPROVED** [review]
LGTM after discussed in CPC meeting.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (tobie) [reply CODE_OF_CONDUCT.md:27]: See proposed CPC charter change in 30239fef21a7a01d0c9595a6b1787def200ebb48.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tobie) [reply CODE_OF_CONDUCT.md:17]: Looking more closely, it seems Appium hasn't updated it CoC in while, so there's more to fix, here: https://github.com/appium/appium/blob/master/CONDUCT.md

**Comment by tobie** [comment]
> The linter is a bit excessive, here, imho, no?

Fixed in a51e8fb562048494f816eed7a5e739cc2d2a0736.

**Comment by mhdawson** [comment]
This was approved in the board meeting on Friday Sep 24 2021

File: .remarkrc [hunk 3]
```diff
@@ -5,6 +5,7 @@
     ["remark-lint-list-item-bullet-indent", false],
     ["remark-lint-no-literal-urls", false],
     ["remark-lint-no-shortcut-reference-link", false],
+    ["remark-lint-no-auto-link-without-protocol", false],
```
Comment (mhdawson) [inline .remarkrc:8]:
```suggestion
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mhdawson) [reply CODE_OF_CONDUCT.md:16]: @tobie I was about to land this but not sure if it should wait on this?

**Comment by mhdawson** [comment]
@tobie I had removed the change to the linter config as I thought that was an accidental inclusion as seems odd to change the linter config as part of this PR. I do see now though that linter fails.

**Comment by Trott** [comment]
> @tobie I had removed the change to the linter config as I thought that was an accidental inclusion as seems odd to change the linter config as part of this PR. I do see now though that linter fails.

`remark-lint-auto-link-without-protocol` is a once-useful now-broken rule that is likely to be removed (if it hasn't already been removed) from `remark-lint` so it's fine to remove/disable it.

Refs: https://github.com/remarkjs/remark/discussions/863#discussioncomment-1383476

**Comment by Trott** [comment]
> > @tobie I had removed the change to the linter config as I thought that was an accidental inclusion as seems odd to change the linter config as part of this PR. I do see now though that linter fails.
> 
> `remark-lint-auto-link-without-protocol` is a once-useful now-broken rule that is likely to be removed (if it hasn't already been removed) from `remark-lint` so it's fine to remove/disable it.
> 
> Refs: [remarkjs/remark#863 (reply in thread)](https://github.com/remarkjs/remark/discussions/863#discussioncomment-1383476)

(Specifically, it will erroneously flag auto-links for email addresses that don't include the `mailto:` protocol, like the one added in this PR.)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mhdawson) [reply CODE_OF_CONDUCT.md:16]: Going to land and it can be fixed up in a follow on PR.

**Pull request merged**